addons/website/static/src/scss/website.edit_mode.scss

```diff
@@ -143,21 +143,15 @@ table.editorbar-panel {
 }
 
 // s_popup in edit mode
-.editor_has_snippets {
-    .s_popup_main {
-        z-index: 1000 !important;
-        left: $o-we-sidebar-width !important;
-        top: var(--o-we-toolbar-height) !important;
-
-        &.s_popup_fixed {
-            .s_popup_full .s_popup_content {
-               left: $o-we-sidebar-width !important;
-            }
+body.editor_enable.editor_has_snippets {
+    .s_popup .modal {
+        top: var(--o-we-toolbar-height);
+        background-color: transparent;
```

Comment:
Feels like it is not necessary, why do you need that?

Reply:
To override [this](https://github.com/odoo/odoo/blob/23b0f38245eced2d40d06e5c3ae2ad5a92d6e396/addons/web_editor/static/src/scss/wysiwyg_snippets.scss#L13) and [this](https://github.com/odoo/odoo/blob/23b0f38245eced2d40d06e5c3ae2ad5a92d6e396/addons/web_editor/static/src/scss/wysiwyg.scss#L429).
Is there a better solution?

Reply:
Mmh indeed... should not this be a bug fix in a previous version then? In all case, add a comment to explain that the background is forced in edition ;)